### DISCUSSION TITLE
Customising the Layer Manager

### DISCUSSION THREAD

**Discussion by debadree25** [GrapesJS/grapesjs#4197] [Q&A] [answered]
First of all, I would like to thank the creator for such an amazing tool it has been immensely helpful for me!

I would like to ask whether it is possible to customize the layer manager just like we can do with the block manager, style manager, etc. I don't seem to find layer manager-specific articles regarding the layer manager in the documentation to any pointers regarding this would be immensely helpful or even pointing out where in the code the native layer manager is implemented would also be helpful!

**Comment by artf** [comment, marked as answer]
Yeah, the LayerManager is one of the modules to refactor, this is why is not yet documented but the idea is to give the same level of customization as other public modules. Here you can find the source of the module: https://github.com/artf/grapesjs/tree/master/src/navigator

**Reply by debadree25** [reply]
Thank you so much! eagerly waiting for that
meanwhile will try to implement it on my own and share any findings here
